specs/_features/eip7805/p2p-interface.md

```diff
@@ -0,0 +1,90 @@
+# EIP-7805 -- Networking
+
+This document contains the consensus-layer networking specification for EIP-7805.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Modifications in EIP-7805](#modifications-in-eip-7805)
+  - [Configuration](#configuration)
+  - [The gossip domain: gossipsub](#the-gossip-domain-gossipsub)
+    - [Topics and messages](#topics-and-messages)
+      - [Global topics](#global-topics)
+        - [`inclusion_list`](#inclusion_list)
+  - [The Req/Resp domain](#the-reqresp-domain)
+    - [Messages](#messages)
+      - [InclusionListByCommitteeIndices v1](#inclusionlistbycommitteeindices-v1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Modifications in EIP-7805
+
+### Configuration
+
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `ATTESTATION_DEADLINE` | `SECONDS_PER_SLOT // 3` | seconds | 4 seconds |
+
+| Name | Value | Description |
+| - | - | - |
+| `MAX_REQUEST_INCLUSION_LIST` | `2**4` (= 16) | Maximum number of inclusion list in a single request |
+| `MAX_BYTES_PER_INCLUSION_LIST` | `2**13` (= 8192) | Maximum size of inclusion list in bytes |
+
+### The gossip domain: gossipsub
+
+#### Topics and messages
+
+The new topics along with the type of the `data` field of a gossipsub message are given in this table:
+
+| Name | Message Type |
+| - | - |
+| `inclusion_list` | `SignedInclusionList` |
+
+##### Global topics
+
+EIP-7805 introduces a new global topic for inclusion lists.
+
+###### `inclusion_list`
+
+This topic is used to propagate signed inclusion list as `SignedInclusionList`.
+The following validations MUST pass before forwarding the `inclusion_list` on the network, assuming the alias `message = signed_inclusion_list.message`:
+
+- _[REJECT]_ The size of `message` is within upperbound `MAX_BYTES_PER_INCLUSION_LIST`.
```

Comment:
Shouldn't this rather check the `transactions` size since the `MAX_BYTES_PER_INCLUSION_LIST` is enforced at the EL level and it doesn't take into consideration the other fields?
```suggestion
- _[REJECT]_ The size of `message. transactions` is within upperbound `MAX_BYTES_PER_INCLUSION_LIST`.
```

Reply:
that's how we implemented it in Lodestar, there was some discussion on it in discord